src/main/java/com/limechain/runtime/hostapi/HostApi.java

```diff
@@ -22,21 +23,29 @@
 @Log
 public class HostApi {
 
+    private static final HostApi INSTANCE = new HostApi();
+
+    private HostApi() {
+        //Singleton
+    }
```

Comment:
This can be an annotation, `@NoArgsConstructor` with the visibility argument set to private